src/linux/backend-v4l2.h

```diff
@@ -159,7 +159,6 @@ namespace librealsense
         };
 
 
-        //Debug Evgeni
```

Comment:
Too bad - this was great legacy knowledge :-)

Reply:
Strange... I don't see it in the local branch

Reply:
Well, you removed it

Reply:
Ok, now I relaize.. :)